content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -0,0 +1,400 @@
+
+This will output the virtual services that are mapped to the waypoint proxy.
+
+### Checking the Waypoint Proxy Traffic
+
+You can use the following command to monitor the waypoint proxy traffic:
+
+{{< text bash >}}
+$ command
+{{< /text >}}
+
+This will output the pods in your cluster. You can then use the `istioctl __` command to get the traffic statistics for each pod.
+
+### Verifying L7 proxy load balancing
+
+
+### Monitoring the AB Deployment and Canary Deployment
+
+You can use the following command to monitor the AB Deployment and Canary Deployment traffic:
+
+{{< text bash >}}
+$ command
+{{< /text >}}
+
+This will output the traffic split configuration for the virtual service.
+
+### Conclusion
+
+This section has described how to monitor the L7 waypoint proxy for the Bookinfo application. For more information on waypoint proxies, please see the Istio documentation.
+
+### Additional Details on Monitoring the Virtual Service Mapping
+
+You can also use the following methods to monitor the virtual service mapping to the waypoint proxy:
+
+* **Use the Istio telemetry dashboards:** The Istio telemetry dashboards provide a graphical view of the traffic flowing through your Istio mesh. You can use these dashboards to monitor the traffic flowing to your waypoint proxies and to identify any problems with the virtual service mapping.
+* **Use Prometheus and Grafana:** You can use Prometheus and Grafana to collect and visualize metrics from your Istio mesh. You can use these tools to monitor the metrics associated with the waypoint proxy configuration and the virtual service mapping.
+
+**Monitoring the Waypoint Configuration for How Those Virtual Services Get Mapped to the Waypoint**
+
+You can use the following methods to monitor the waypoint configuration for how virtual services get mapped to the waypoint:
+
+* **Use the Istio telemetry dashboards:** The Istio telemetry dashboards provide a graphical view of the traffic flowing through your Istio mesh. You can use these dashboards to monitor the traffic flowing to your waypoint proxies and to identify any problems with the virtual service mapping.
+* **Use the Istio `istioctl` command:** You can use the Istio `istioctl` command to view the waypoint configuration. This configuration includes the virtual services that are mapped to the waypoint.
+* **Use Prometheus and Grafana:** You can use Prometheus and Grafana to collect and visualize metrics from your Istio mesh. You can use these tools to monitor the metrics associated with the waypoint proxy configuration and the virtual service mapping.
+
+By monitoring the waypoint proxy and the virtual service mapping, you can ensure that your Istio mesh is operating as expected.
+
+## L7 Authorization Policy
+TODO
+
+## Monitoring and Telemetry with L7 Waypoint Proxy
+TODO
+
+## How to use the Waypoint proxy for hairpinning
+TODO
+
+## Co-existence of Ambient/ L7 with Side car proxies
+How to use the Waypoint proxy with sidecar proxies
+TODO
+
```

Comment:
do the following examples have anything unique to waypoint or are they just additional examples?  I think they may just work the same way so we don't really need to include them